### PR TITLE
Update plugin description & replace 'subscription' with 'membership' throughout readme. #16

### LINKED ISSUE #16: Update plugin description
Current:

> Control the number of times a subscription can be purchased.

Proposed:

> Control the number of times a membership level can be purchased.

### PR DESCRIPTION
**Pull request by nosegraze**
Closes #16

### CODE REVIEW

File: settings/subscription-levels.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ function subscription_form_fields( $level ) {
 
 		<td>
 			<input type="number" id="rcp-lqa-total-available" name="rcp-lqa-total-available" value="<?php echo esc_attr( $qty ); ?>" style="width: 60px;">
-			<p class="description"><?php _e( 'Limit the total number of times this subscription level can be sold. For example, enter 20 to allow a maximum of 20 sales. After 20 sales, the subscription level will not show on your registration form. Renewal payments do not count towards total sales. Enter 0 for unlimited. If you set up limited quantities on a subscription level that already has sales, those sales will count towards the total.', 'rcp-lqa' ); ?></p>
+			<p class="description"><?php _e( 'Limit the total number of times this sembership level can be sold. For example, enter 20 to allow a maximum of 20 sales. After 20 sales, the sembership level will not show on your registration form. Renewal payments do not count towards total sales. Enter 0 for unlimited. If you set up limited quantities on a sembership level that already has sales, those sales will count towards the total.', 'rcp-lqa' ); ?></p>
```
Comment (misulicus) [inline settings/subscription-levels.php]: I think this is a type.
Several places in this sentence: `sembership ` should be `membership `

File: settings/subscription-levels.php [hunk 2]
```diff
@@ -11,7 +11,7 @@
 defined( 'WPINC' ) or die;
 
 /**
- * Displays the quantity available form field on the subscription level edit screen.
+ * Displays the quantity available form field on the sembership level edit screen.
```
Comment (misulicus) [inline settings/subscription-levels.php]: `sembership` to `membership`

File: settings/subscription-levels.php [hunk 3]
```diff
@@ -43,10 +43,10 @@ function subscription_form_fields( $level ) {
 add_action( 'rcp_add_subscription_form', '\RCP\Addon\LimitedQuantityAvailable\Settings\subscription_form_fields' );
 
 /**
- * Saves the quantity available defined on the subscription edit screen.
+ * Saves the quantity available defined on the sembership level edit screen.
```
Comment (misulicus) [inline settings/subscription-levels.php]: `sembership` to `membership`

File: settings/subscription-levels.php [hunk 4]
```diff
@@ -43,10 +43,10 @@ function subscription_form_fields( $level ) {
...
  *
  * @param int   $id   ID of the level being added or edited.
- * @param array $args Arguments for updating the subscription level.
+ * @param array $args Arguments for updating the sembership level.
```
Comment (misulicus) [inline settings/subscription-levels.php]: `sembership` to `membership`

File: settings/subscription-levels.php (hunk 1 above)
Comment (nosegraze) [reply settings/subscription-levels.php]: That's what I get for copying and pasting the _wrong_ thing!

**Pull request merged**